src/main/java/com/techconative/restel/core/model/comparator/StringJsonComparator.java

```diff
@@ -0,0 +1,15 @@
+package com.techconative.restel.core.model.comparator;
```

Comment:
This can be in a different package. For ex, `com.techconative.restel.extension.jsoncompare.comparator`. The entities in `core.model.comparator` are the core comparators which does the actual verification of the response/headers. Whereas this class is used by the core comparators.